dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/lb/RoundRobinSelection.java

```diff
@@ -48,7 +48,7 @@ public RoundRobinSelection() {
 	}
 
 	@Override
-	public HostConnectionPool<CL> getPoolForOperation(BaseOperation<CL, ?> op) throws NoAvailableHostsException {
+	public HostConnectionPool<CL> getPoolForOperation(BaseOperation<CL, ?> op, String hashtag) throws NoAvailableHostsException {
```

Comment:
What does the new hashtag parameter do here in this function? Why did you add it?

Reply:
Currently nothing. If an app uses `RoundRobinSelection` then hashtags are of no meaning. I could probably through an exception if the Dynomite cluster has a hashtag defined and the app uses `RoundRobinSelection`. What do you think?

Reply:
no, why would you throw an error for RoundRobin with hashtags?

Reply:
Because in RoundRobin you are not using the keys to determine where the data are going to be sent, so hashtags as part of the key have no meaning. In RoundRobin, Dyno uses a `circularList`. Now let us assume that sb configures Dynomite to support hashtags, then Dyno and Dynomite would be misaligned.

Reply:
not sure what you mean here. but that's ok.